dronegen/buildbox.go

```diff
@@ -58,19 +58,19 @@ func buildboxPipelineStep(buildboxName string, fips bool) step {
 			`chown -R $UID:$GID /go`,
 			`docker login -u="$$QUAYIO_DOCKER_USERNAME" -p="$$QUAYIO_DOCKER_PASSWORD" quay.io`,
 			fmt.Sprintf(`make -C build.assets %s`, buildboxName),
-			fmt.Sprintf(`docker push quay.io/gravitational/teleport-%s:$RUNTIME`, buildboxName),
+			fmt.Sprintf(`docker push quay.io/gravitational/teleport-%s:$BUILDBOX_VERSION`, buildboxName),
 		},
 	}
 }
 
 func buildboxPipeline() pipeline {
 	p := newKubePipeline("build-buildboxes")
 	p.Environment = map[string]value{
-		"RUNTIME": goRuntime,
-		"UID":     {raw: "1000"},
-		"GID":     {raw: "1000"},
+		"BUILDBOX_VERSION": buildboxVersion,
+		"UID":              {raw: "1000"},
+		"GID":              {raw: "1000"},
 	}
-	p.Trigger = triggerPushMasterOnly
+	p.Trigger = pushTriggerFor("master", "branch/v9")
```

Comment:
Given that this change is only on master, do we need branch/v9 here?  Wouldn't that `branch/v9` be needed in the v9 port of this PR?

Alternatively, can we wildcard `branch/*` here for port and future branch proofing?